dap_job_quality/pipeline/keyword_search.py

```diff
@@ -0,0 +1,90 @@
+"""'
+This script creates an initial keyword search by sentence for terms related to job quality data, and outputs a dataframe with each sentence, whether a benefit keyword was found, and whether a harm keyword was found, and any terms found.
+Because we're working through the files in google sheets to start with, it reads in and outputs data locally.
+So to update the data, first download the current search terms, and place in the /inputs folder.
+
+To run this script to generate 10,000 sentences of data, run the following command from the root directory:
+
+python dap_job_quality/pipeline/keyword_search.py
+
+It will output a file into outputs/data, which can be manually uploaded to google sheets for further analysis."""
+
+from dap_job_quality import PROJECT_DIR, logger
+from dap_job_quality.getters.ojo_getters import get_ojo_sample
+from dap_job_quality.utils.text_cleaning import clean_text, split_sentences
+import pandas as pd
+
+
```

Comment:
Just a minor style thing - could you add constant up at the top here - eg the number of sentences; input and output file paths. Something like this:
```
NO_SENTENCES = 10000
INPUT_PATH = PROJECT_DIR / "inputs/keyword_lookup.csv"
todays_date = pd.to_datetime("today").date()
OUTPUT_PATH = PROJECT_DIR / f"outputs/data/keyword_search_{todays_date}.csv"
```